glue/tests/helpers.py

```diff
@@ -66,6 +72,12 @@ def make_skipper(module, label=None, version=None):
 requires_qt = pytest.mark.skipif(str(not QT_INSTALLED),
                                  reason='An installation of Qt is required')
 
+PYQT_GT_59, _ = make_skipper('PyQt5', version='5.10')
+
+REQUIRES_PYQT_GT_59 = PYQT_GT_59 and not PYSIDE2_INSTALLED
+
+requires_pyqt_gt_59 = pytest.mark.skipif(str(not REQUIRES_PYQT_GT_59), reason='Requires PyQt > 5.9')
```

Comment:
```suggestion
requires_pyqt_gt_59_or_pyside2 = pytest.mark.skipif(str(not PYQT_GT_59 and not PYSIDE2_INSTALLED),
                                                    reason='Requires PyQt > 5.9 or PySide2')
```
The suggestion probably came around backwards as I was thinking from the `skipif` side... The point anyway was _not_ to skip it for the `pysideXX` jobs either, which hopefully this construct should achieve.

Reply:
Oh, yeah, that makes more sense :)